safe-global/safe-wallet-web
# Test: add tx signing for NFTs

## What it solves
Solves a task for automation to add a signing action to the Send NFT transaction

## How this PR fixes it
Created a new safe and added an NFT to it
Selects the NFT, sets a high nonce so there is no risk of execution
Signs the tx
Checks that the tx is queued with the name of the NFT in said transaction

## How to test it
Run the test "Verify Send NFT transaction has been created" and check it passes
Also no other test should fail

## Checklist
